js/ui/file_manager/ui.file_manager.js

```diff
@@ -364,9 +364,9 @@ class FileManager extends Widget {
             },
 
             /**
-            * @name dxTreeViewOptions.customThumbnail
+            * @name dxFileManagerOptions.customThumbnail
```

Comment:
customThumbnail -> customizeThumbnail?
/cc @RomanTsukanov @TikhonovaTatyana

Reply:
We gotta discuss the names of all the API members

Reply:
Of course, we'll prepare full list of public API and discuss with tech writers. At now, we constantly change public API therefore better choice is discuss it later.